packages/main/src/plugin/container-registry.ts

```diff
@@ -435,7 +435,6 @@ export class ContainerProviderRegistry {
           }
 
           // if we have a libpod API, grab containers using Podman API
-          // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
It's not needed here as type is defined for the `containers` variable. I believe this is legacy